app/models/concerns/hud_concerns/client.rb

```diff
@@ -45,6 +45,16 @@ def gender_multi
       end
     end
 
+    def race_multi
+      ret = []
+      HudUtility2024.race_field_name_to_id.except(:RaceNone).each do |k, v|
```

Comment:
I'm probably missing something, but this is a hash keyed on an integer, so I think `except(:RaceNone)` should be removed or replaced with `except(8, 9, 99)`

Reply:
Huh, I guess I'm wrong.